src/network/axiosInstance.ts

```diff
@@ -0,0 +1,26 @@
+import axios from 'axios';
+
+const API_VERSION = 3;
+
+const ACCESS_TOKEN =
+  'eyJhbGciOiJIUzI1NiJ9.eyJhdWQiOiI2OWNlNzI5ZGNjYjc1Nzk4MTU5YmQ1NTc1MjUzNTI0MiIsIm5iZiI6MTcyOTc0NjY2NC42NzEyNjUsInN1YiI6IjY3MTljZjRkNWJlOWU4NzU5ZGE2ZGQxYiIsInNjb3BlcyI6WyJhcGlfcmVhZCJdLCJ2ZXJzaW9uIjoxfQ.CRaTvR-R67HVxts401F1CIIbfbWKPlJjRQyHjbUw6ME';
```

Comment:
haven't removed base url and access token from here?

Reply:
I pushed the changes, could be the previous file.